server/routes/task.js

```diff
@@ -3,26 +3,26 @@ const { body, validationResult } = require("express-validator");
 const router = express.Router();
 const Task = require("../models/Task");
 const fetchUser = require("../middleware/fetchUser");
-const authAdmin=require('../middleware/authAdmin')
+const authAdmin = require("../middleware/authAdmin");
+const User = require("../models/User");
 
 router.post(
   "/createTask/:id",
   [
-    body("title", "Title must be 5 characters long").isLength({ min: 5 }),
-    body("description").isLength({ min: 5 }),
+    body("title").isLength({ min: 5 }).withMessage("Title can not be empty"),
+    body("description").isLength({ min: 5 }).withMessage("Description can not be empty"),
   ],
   authAdmin,
   async (req, res) => {
     const errors = validationResult(req);
-    console.log(req.body.title)
-    console.log(req.body.description)
+    console.log(req.body.title);
```

Comment:
unsual changes, if you don't put semi colon still this code will work.
its just a distraction in git changes, try to commit only nessacary changes.